packages/import-utils/src/create-sandbox/templates.ts

```diff
@@ -90,6 +90,10 @@ export function getTemplate(
     return "apollo";
   }
 
+  if (totalDependencies.indexOf("ember-cli") > -1) {
+    return "node";
```

Comment:
I don't see any real benefit to defining a new template type. Ember-CLI apps are a subset of node apps.

Reply:
You have a good point! I wonder how we should handle this, because everything server really is just node. We initially made it non-node templates so we can have the option to have custom logic in the future and to make it easier for people to color code what they're working on.

But I agree that this is a convoluted concept, we should find a better way to handle templates. I just haven't found a good one yet, maybe just color coding by dependency types? That's what we currently do for server templates and it definitely helps, but it still might be a bit confusing to see `ember` as a template while it really is `node`.

I think that we should initially make this `ember` template, we can find a proper solution for this in the future.

Reply:
@CompuIves done. I'm calling the template `"ember-cli"` because this build tool is used for several things that aren't ember.js (i.e., [Glimmer.js](https://glimmerjs.com/))